app/src/main/java/org/piramalswasthya/cho/repositories/ProcedureRepo.kt

```
@@ -1,20 +1,163 @@
+        benVisitNo: Int
+    ): List<ProcedureDataWithComponent> {
         return withContext(Dispatchers.IO) {
             procedureDao.getProceduresWithComponent(patientID, benVisitNo)
         }
     }
+
+    suspend fun pullLabProcedureMasterData(): Boolean {
+        return try {
+            getProcedureMasterData();
+            true
+        } catch (e: Exception) {
```

detekt

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)